framework/DataObjects/XDataSet.py

```diff
@@ -783,7 +783,7 @@ def _collapseNDtoDataArray(self,data,var,labels=None,dtype=None):
       dataType = dtype
     # method = 'once' # see below, parallelization is possible but not implemented
     # first case: single entry per node: floats, strings, ints, etc
-    if isinstance(data[i],(float,str,unicode,int,bool,np.bool_)):
+    if isinstance(data[i],(float,str,int,unicode,bool,np.bool_, np.integer)):
```

Comment:
why was this modification needed?

Reply:
We need np.integer, because the type np.int32, np.int64 will not catch by previous statement.